File: app/src/main/java/com/semanienterprise/softcom/models/FormData.java
package com.semanienterprise.softcom.models;

import java.io.Serializable;
import java.util.List;
import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class FormData implements Serializable {
	@SerializedName("pages")
	private List<PagesItem> pages;

    @SerializedName("name")
	private String name;

	@SerializedName("id")
	private String id;

    @Override
 	public String toString(){
		return
			"FormData{" +
			"pages = '" + pages + '\'' +
			",name = '" + name + '\'' +
			",id = '" + id + '\'' +
			"}";
		}
}

File: app/src/main/java/com/semanienterprise/softcom/models/FormData.kt
package com.semanienterprise.softcom.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class FormData : Serializable {

    @SerializedName("pages")
    var pages: List<PagesItem>? = null

    @SerializedName("name")
    var name: String? = null

    @SerializedName("id")
    var id: String? = null

    override fun toString(): String {
        return "FormData{" +
                "pages = '" + this.pages + '\''.toString() +
                ",name = '" + this.name + '\''.toString() +
                ",id = '" + this.id + '\''.toString() +
                "}"
    }
}